oldScripts/gradescope_to_spreadsheet.py

```diff
@@ -41,11 +43,14 @@
 # The ASSIGNMENT_ID constant is for users who wish to generate a sub-sheet (not update the dashboard) for one assignment, passing it as a parameter.
 ASSIGNMENT_ID = (len(sys.argv) > 1) and sys.argv[1]
 
+load_dotenv()
```

Comment:
We will need to add instructions in a README on what configurations should go in the `.env` file if users want to run this script. This can be a separate ticket.

Reply:
Noted!